octokit/resources.py

```diff
@@ -185,4 +185,4 @@ def fetch_resource(self, method, *args, **kwargs):
 
     schema = self.fetch_schema(req)
     resource = Resource(self.session, schema=schema, url=url, name=humanize(self.name))
-    return resource
+    return resource
```

Comment:
*nit -- newline at the end of file to keep it consistent with the rest. I think you can make sublime do this automatically